### PR TITLE
Added driver model

### PR DESCRIPTION
**Pull request by domenic99**
#12 Driver model

### CODE REVIEW

File: db/migrate/20161206131603_create_drivers.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateDrivers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :drivers do |t|
+      t.string :name
+      t.string :phone
```
Comment (rakvium) [inline db/migrate/20161206131603_create_drivers.rb]: @domenic99 are you sure that we could have any drivers without a phone in our database?

**Comment by rakvium** [comment]
@domenic99 by the way, why don't you use Devise for drivers authentication?

**Comment by domenic99** [comment]
Check out my fixes )

File: test/fixtures/drivers.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+one:
+  name: MyString
+  phone: MyString
+  pass: MyString
+  auto: MyString
+  status: MyString
+
+two:
+  name: MyString
+  phone: MyString
```
Comment (rakvium) [inline test/fixtures/drivers.yml]: This fixture won't work accordingly to https://github.com/rakvium/taxi-api/pull/18/files#diff-a64c19699f111b607af52963626e2d83R13

Comment (domenic99) [inline test/fixtures/drivers.yml]: Changed to another value. Is it a correct fix ?

**Pull request merged**